analyze-ms-stats/environment.yml

```diff
@@ -0,0 +1,55 @@
+name: ms-stats
```

Comment:
I don't think we'll need a new environment for this analysis, but let's defer that decision for when we integrate this with a scheduled workflow.  It's okay to commit this temporarily as a record of what packages are known to be compatible with the code.

Reply:
That sounds good to me!